app/pages/analyst/dashboard.tsx

```diff
@@ -94,9 +96,18 @@ const AnalystDashboard = ({
         orderBy: null,
       },
     };
+    cookie.remove('analyst.sort');
     router.replace(url, url, { shallow: true });
   };
 
+  useEffect(() => {
+    const orderByParam = cookie.get('analyst.sort');
+    if (orderByParam)
```

Comment:
I'm tempted to ask for a condition here with `orderByParam !== 'undefined'` here to prevent any errors there, but the code is working well now. Up to you if you want to add it

Reply:
Oh sorry this was auto merged though I can make a new small PR though shouldn't the `if (orderByParam)` cover that?

Reply:
Yeah I noticed, I think it's okay to leave out since the likelihood of it happening is slim since we've put in that check for setting that sort order cookie.